developer-model-service/catalog-info.yaml

```diff
@@ -0,0 +1,2036 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: developer-model-service
+  annotations:
+    backstage.io/techdocs-ref: dir:./
+  description: 'vLLM and 3scale-based model service providing models for developer tools.'
+  links:
+    - url: https://model-service.apps.domain.com
+      title: Access
+      type: website
+      icon: WebAssett
+    - url: https://ibm-granite-8b-code-instruct-vllm.apps.domain.com
+      title: API URL
+      type: website
+      icon: WebAsset
+  tags:
+    - genai
+    - ibm-granite
+    - vllm
+    - llm
+    - developer-model-service
+    - authenticated
+    - gateway
+spec:
+  type: model-server
+  owner: user:johnmcollier
+  lifecycle: production
+  providesApis:
+    - model-service-api
+  dependsOn:
+    - resource:ibm-granite-8b-code-instruct
+    - api:model-service-api
+  profile:
+    displayName: "Developer Model Service"
+---
+####################################
+# RESOURCES
+# A resource (with type `ai-model`) represents an AI model that's deployed on an associated model server
+####################################
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: ibm-granite-8b-code-instruct
+  annotations:
+    backstage.io/techdocs-ref: dir:ibm-granite-8b-code-instruct
+  description: 'IBM Granite is a decoder-only code model for code generative tasks (e.g. fixing bugs, explaining code, documenting code. Trained with code written in 116 programming languages.'
+  links:
+    - url: https://model-service.apps.domain.com
+      title: Access
+      type: website
+      icon: WebAssett
+    - url: https://ibm-granite-8b-code-instruct-vllm.apps.domain.com
+      title: API URL
+      type: website
+      icon: WebAsset
+    - url: https://huggingface.co/ibm-granite/granite-8b-code-instruct
+      title: Model Repository
+      type: website
+      icon: WebAsset
+    - url: https://huggingface.co/ibm-granite/granite-8b-code-instruct-4k/tree/main
+      title: Download Model Artifacts
+      type: website
+      icon: WebAsset
+    - url: https://www.apache.org/licenses/LICENSE-2.0
+      title: Apache-2.0 License
+      type: website
+      icon: WebAsset
+  tags:
+    - genai
+    - ibm
+    - llm
+    - granite
+    - conversational
+    - task-text-generation
+spec:
+  type: ai-model
+  owner: user:johnmcollier
+  lifecycle: production
+  providesApis:
+    - model-service-api
+  dependencyOf: 
+    - component:developer-model-service
+  profile:
+    displayName: "IBM Granite Code Model"
+---
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: model-service-api
+  title: Model Service API
+  description: 'The OpenAI-compatible REST API exposed by the vLLM-based developer model service.'
+  tags:
+    - api
+    - openai
+    - vllm
+  links:
+    - url: https://model-service.apps.domain.com/docs
+      title: Model Service API Documentation
+      type: website
+      icon: WebAsset
+spec:
+  type: openapi
```

Comment:
and this per your google doc is:

```
apiSchema | Json | The API schema for the model server (e.g. OpenAPI, Swagger, etc)
```

Reply:
API `spec.type` is fixed to whichever format the API definition uses (e.g. `OpenAPI`, `asyncapi`, `graphql`, `grpc`, per [this doc](https://backstage.io/docs/features/software-catalog/descriptor-format/#spectype-required-2))

Reply:
The API definition itself (the `apiSchema` you're referring to here), is under `spec.definition`, e.g. https://github.com/redhat-ai-dev/model-catalog/blob/6b67aae6d9cc71f1354b2a3df29940cfc99e99ba/developer-model-service/catalog-info.yaml#L113-L2036